account-compression/sdk/src/events/index.ts

```diff
@@ -46,3 +49,70 @@ export function deserializeApplicationDataEvent(
       throw Error('Unable to decode buffer as ApplicationDataEvent');
   }
 }
+
+/**
+ * Helper function to extract the ChangeLogEvent V1 from a TransactionResponse
+ * @param txResponse - TransactionResponse from the `@solana/web3.js`
+ * @param programId - PublicKey of the program (aka `programId`) that utilized the leaf on the tree
+ * @param noopProgramId - program id of the noop program used (default: `SPL_NOOP_PROGRAM_ID`)
+ * @returns
+ */ 
+export function getChangeLogEventV1FromTransaction(
+  txResponse: TransactionResponse,
+  programId: PublicKey,
+  noopProgramId: PublicKey = SPL_NOOP_PROGRAM_ID,
+){
+  // ensure a transaction response was provided
+  if (!txResponse) throw Error("No txResponse provided");
+
+  // find the correct index of the `programId` instruction
+  const relevantIndex =
+  txResponse.transaction.message.compiledInstructions.findIndex(
+    (instruction) => {
+      return (
+        txResponse?.transaction.message.staticAccountKeys[
+          instruction.programIdIndex
+        ].toBase58() === programId.toBase58()
```

Comment:
Make sure to support transactions that use look up tables
```typescript
let accounts = staticAccountKeys + readOnly + writable;
if (accounts[instruction.programIdIndex] == programId.toBase58()) {
  ...
}
```

Reply:
This is now added into the PR